python_file/project_note_files.py

```diff
@@ -0,0 +1,138 @@
+"""Проект з керуванням файлами
+Простий нотатник
+Мета: Створити програму для ведення нотаток з базовим керуванням файлами.
+    Функції:
+Створювати та зберігати текстові нотатки у файлах
+Читати існуючі нотатки
+Перераховувати доступні нотатки
+Шукати нотатки за ключовими словами
+    Фокус реалізації:
+Використовувати операції з файлами для читання та запису
+Створювати відповідні угоди з іменування файлів
+Реалізувати керування директоріями
+Граціозно обробляти помилки файлів
+Інтерфейс користувача: Інтерфейс командного рядка для створення, 
+перегляду, переліку та пошуку нотаток.
+    Виклики:
+Реалізувати надійне збереження та завантаження файлів
+Створити організовану структуру файлів
+Обробляти потенційні помилки файлів
+Реалізувати ефективний пошук нотаток"""
+
+from pathlib import Path
+from datetime import datetime
+
+# Абсолютний шлях до базової папки
+base_dir = Path(r"C:/Users/assto/PythonTasksProject/2_Git_Theory/python_file")
```

Comment:
Don't hardcode your system path into codebases. It's a bad practice and it guaranties the possibility of bugs.
```suggestion
base_dir = Path(__file__).resolve().parent
```